### ISSUE TITLE
NullPointerException when calling Meta.apply() on an H2 database with views

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10205]
When using an existing H2 JDBC connection on a database containing views, and then calling this:

```java
System.out.println(ctx.meta().apply("drop schema INFORMATION_SCHEMA cascade").getSchemas());
```

Then, there's a NPE:

```
16:56:18,600 DEBUG [org.jooq.impl.Interpreter                         ] - create view "CASE sensitive SCHEMA with SPACES"."Case Sensitive View"("X x")
as
Exception in thread "main" java.lang.NullPointerException
	at org.jooq.impl.Interpreter.accept0(Interpreter.java:802)
	at org.jooq.impl.Interpreter.accept(Interpreter.java:188)
	at org.jooq.impl.InterpreterMetaProvider.provide(InterpreterMetaProvider.java:94)
	at org.jooq.impl.DefaultDSLContext.meta(DefaultDSLContext.java:470)
	at org.jooq.impl.AbstractMeta.apply(AbstractMeta.java:340)
	at org.jooq.impl.AbstractMeta.apply(AbstractMeta.java:325)
	at JDBC.main(JDBC.java:17)
```

This is because we don't have the source code for these views yet: #9818, but it should still not produce a NPE

**Comment by lukaseder** [comment]
Fixing this leads to another exception:

```
org.jooq.exception.DataDefinitionException: Primary key does not exist
	at org.jooq.impl.Interpreter.primaryKeyNotExists(Interpreter.java:1091)
	at org.jooq.impl.Interpreter.addForeignKey0(Interpreter.java:372)
	at org.jooq.impl.Interpreter.addForeignKey(Interpreter.java:327)
	at org.jooq.impl.Interpreter.addConstraint(Interpreter.java:745)
	at org.jooq.impl.Interpreter.accept0(Interpreter.java:536)
	at org.jooq.impl.Interpreter.accept(Interpreter.java:181)
	at org.jooq.impl.InterpreterMetaProvider.provide(InterpreterMetaProvider.java:94)
	at org.jooq.impl.DefaultDSLContext.meta(DefaultDSLContext.java:470)
	at org.jooq.impl.AbstractMeta.apply(AbstractMeta.java:340)
	at org.jooq.impl.AbstractMeta.apply(AbstractMeta.java:325)
	at org.jooq.test.all.testcases.MetaDataTests.testMetaApply(MetaDataTests.java:791)
	at org.jooq.test.jOOQAbstractTest.testMetaApply(jOOQAbstractTest.java:2243)
```

**Comment by lukaseder** [comment]
That second issue happens because of a test case from https://github.com/jOOQ/jOOQ/issues/5972:

```sql
create table "PUBLIC"."T_5972_REFERENCED" (
  "A" integer not null,
  "B" integer null,
  constraint "PK_T5972_REFERENCED"
    primary key ("A")
);

alter table "PUBLIC"."T_5972_REFERENCING"
  add constraint "FK_T5972_REFERENCED"
    foreign key (
      "A",
      "B"
    )
    references "PUBLIC"."T_5972_REFERENCED" (
      "A",
      "B"
    );
create unique index "PUBLIC"."PRIMARY_KEY_8" on "PUBLIC"."T_5972_REFERENCED"("A" asc);
create unique index "PUBLIC"."UK_T5972_REFERENCED_INDEX_8" on "PUBLIC"."T_5972_REFERENCED"(
  "A" asc,
  "B" asc
);
```

The order of creation of indexes and foreign keys should probably be reversed in `Meta.ddl()`

**Comment by lukaseder** [comment]
I've created a separate issue for that latter case: #10206

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.14.0. Backport scheduled for 3.13.3 (#10208)
